test/TEST-30-ONCLOCKCHANGE/test.sh

```diff
@@ -29,6 +29,12 @@ test_setup() {
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-resolved.service
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-machined.service
 
+        # extend the watchdog
+        cat >$initdir/etc/systemd/system/testsuite.service <<EOF
+[Service]
+WatchdogSec=10min
+EOF
```

Comment:
hmm, wah? we write this service file again a few lines further down?

Reply:
ouch. fixed...